addons/sys_rack/sem90/fnc_sem90MountableRadio.sqf

```diff
@@ -20,7 +20,7 @@
 
 params ["_rackId", "", "_eventData", ""];
 
-if (([_rackId] call FUNC(getMountedRadio)) != "") exitWith {false}; // If a radio is already mounted we can't mount another.
+if !(([_rackId] call FUNC(getMountedRadio)) isEqualTo "") exitWith {false}; // If a radio is already mounted we can't mount another.
```

Comment:
This is technically slightly slower, 2 commands instead of 1 (`isEqualTo` + `!`). Only advantage is it won't throw an error if types don't match, but I think we should prefer if it does in this case.

It's also longer. I wouldn't change this.